src/main/java/com/matuageorge/technicalmaintenanceschedule/service/UserServiceImpl.java

```diff
@@ -48,6 +48,7 @@ public UserDto save(UserDto userDto) throws ValidationException, ResourceAlready
 
     @Override
     public UserDto update(Long userId, UserDto userDto) {
+        userRepository.findById(userId);
```

Comment:
Why? What is the purpose of this code? If you want to check that user exists, you should check if you find user in db